doc/design/vic-machine/update.md

```diff
@@ -0,0 +1,175 @@
+# VCH Reconfigure (P0)
+This document defines the VCH reconfigure command line, and the related changes in vic-machine, to make VCH reconfigure process more reasonable.
+
+Here is the key points considerred in this Definition
+- Batch Operation
+
+  We'd like to change VCH configurations more than one item at one time to make the command easy to use
+- Reuse all options used in vic-machine create
+
+  In vic-machine create, we already have 57 options to define VCH configurations, though not all of them can be updated, quite amount of them should be able to be changed, and this number will grow up over time, so we definitely do not want to introduce 3X that number of options to reconfigure (update means add/modify/remove)
+
+## Options
+First option is to reconfigure all in one command
+
+### Implicit Add/Modify and Explicit Delete
+All existing vic-machine create options are supported in vic-machine reconfigure command. The meaning of those options are as following:
+- for elements with labels (e.g. container-network, volume-store), if there's a value provided against that label in the reconfigure, it's a modify operation
+- for elements without labels (e.g. --cert), if there's a value provided it's a replacement
+- if need to remove existing configuration (e.g. static ip configuration for client-network, or registry-ca), define new options as --clear-registry-cas, --clear-client-network. Then existing configuration for registry-ca and/or client-network will be cleared. If not new configuration e.g. for client-network specified, it will share with other networks with same logic in vic-machine create.
```

Comment:
cause sometimes we use several options to define one configuration, e.g. static ip for any network. In that case, one clear option is enough. So we don't need that massive number of clear options, but certainly, that will be a lot.

But this clear option is introduced for delete existing configuration, empty option cannot solve this issue for list options. User might want to add a few into existing list, or remove all existing values, and add another few values.
For the latter case, empty option does not help